public/sideeffect.php

```diff
@@ -0,0 +1,8 @@
+<?php
+require ('./review.php');
```

Comment:
__[code_sniffer]__
* Space before opening parenthesis of function call prohibited

<!-- Comment by [SideCI](https://sideci.com) -->